akka-actor/src/main/scala/akka/util/OptionVal.scala

```diff
@@ -9,6 +9,7 @@ package akka.util
 private[akka] object OptionVal {
 
   def apply[A >: Null](x: A): OptionVal[A] = new OptionVal(x)
+  def fromNullable[A >: Null](x: A): OptionVal[A] = new OptionVal(x)
```

Comment:
Actually I thought I needed this but don't, will remove.
Sorry, PR is not completely cleaned up, was so happy it actually worked last night when fixed ByteOrder issue last night that pushed and went to bed ;)